staticcheck.conf

```diff
@@ -0,0 +1,6 @@
+checks = [
+	"inherit",
+	"-SA5008",  # duplicate struct tag (cli/cmd_*.go use this for "choice")
+	"-ST1012",  # error var should have name of the form errFoo
```

Comment:
Out of curiosity, what is the rationale for disabling this one?

Reply:
Because these "errors" were actually not errors but sentinel values, so shouldn't really be named `errFoo`. However, I think this is a reasonable rule in general, so I ended up using line-ignores for those and re-enabling this rule in staticcheck.conf -- thanks.